diff --git a/app/menu.std.ts b/app/menu.std.ts
index dcd65c0..b0ae4d5 100644
--- a/app/menu.std.ts
+++ b/app/menu.std.ts
@@ -235,12 +235,10 @@ export const createTemplate = (
         });
       }
 
-      if (options.development) {
         fileMenu.submenu.unshift({
           label: i18n('icu:menuSetupAsStandalone'),
           click: setupAsStandalone,
         });
-      }
 
       fileMenu.submenu.unshift({
         type: 'separator',
diff --git a/package.json b/package.json
index 31399e6..1896916 100644
--- a/package.json
+++ b/package.json
@@ -527,7 +527,7 @@
         }
       },
       "target": [
-        "deb"
+        "rpm"
       ],
       "icon": "build/icons/png",
       "publish": [],
diff --git a/ts/state/ducks/app.preload.ts b/ts/state/ducks/app.preload.ts
index ddda047..e2e4398 100644
--- a/ts/state/ducks/app.preload.ts
+++ b/ts/state/ducks/app.preload.ts
@@ -94,7 +94,6 @@ function openStandalone(): ThunkAction<
 > {
   return dispatch => {
     if (getEnvironment() === Environment.PackagedApp) {
-      return;
     }
 
     window.IPC.addSetupMenuItems();
