diff --git a/package.json b/package.json
index 75f277b..0fdc288 100644
--- a/package.json
+++ b/package.json
@@ -558,7 +558,7 @@
       },
       "artifactName": "${name}_${version}_${arch}.${ext}",
       "target": [
-        "deb"
+        "rpm"
       ],
       "icon": "build/icons/png",
       "publish": [
diff --git a/ts/components/conversation/Message.dom.tsx b/ts/components/conversation/Message.dom.tsx
index 1f99d5c..a9efab9 100644
--- a/ts/components/conversation/Message.dom.tsx
+++ b/ts/components/conversation/Message.dom.tsx
@@ -3459,7 +3459,7 @@ export class Message extends React.PureComponent<Props, State> {
     const { expired, expiring, isTargeted, imageBroken } = this.state;
 
     if (expired) {
-      return null;
+      // Avoid deleting expired messages from UX
     }
 
     if (isSticker && (imageBroken || !attachments || !attachments.length)) {
@@ -3560,7 +3560,6 @@ export class Message extends React.PureComponent<Props, State> {
             shouldCollapseAbove && 'module-message--collapsed-above',
             shouldCollapseBelow && 'module-message--collapsed-below',
             isTargeted ? 'module-message--targeted' : null,
-            expiring ? 'module-message--expired' : null
           )}
           data-testid={timestamp}
           tabIndex={0}
diff --git a/ts/services/expiringMessagesDeletion.preload.ts b/ts/services/expiringMessagesDeletion.preload.ts
index c95a8d9..45b4ed5 100644
--- a/ts/services/expiringMessagesDeletion.preload.ts
+++ b/ts/services/expiringMessagesDeletion.preload.ts
@@ -25,6 +25,8 @@ class ExpiringMessagesDeletionService {
   }
 
   async #destroyExpiredMessages() {
+    // Avoid deleting expired messages from DB
+    return;
     try {
       log.info('destroyExpiredMessages: Loading messages...');
       const messages = await DataReader.getExpiredMessages();
@@ -68,6 +70,8 @@ class ExpiringMessagesDeletionService {
   }
 
   async #checkExpiringMessages() {
+    // Avoid deleting expired messages from DB
+    return;
     log.info('checkExpiringMessages: checking for expiring messages');
 
     const soonestExpiry = await DataReader.getSoonestMessageExpiry();
diff --git a/ts/util/deleteForEveryone.preload.ts b/ts/util/deleteForEveryone.preload.ts
index 2e6ba1d..b602c65 100644
--- a/ts/util/deleteForEveryone.preload.ts
+++ b/ts/util/deleteForEveryone.preload.ts
@@ -13,6 +13,7 @@ import { isAciString } from './isAciString.std.ts';
 import { eraseMessageContents } from './cleanup.preload.ts';
 import { notificationService } from '../services/notifications.preload.ts';
 import { DataWriter } from '../sql/Client.preload.ts';
+import { postSaveUpdates } from '../util/cleanup.preload.js';
 
 const log = createLogger('deleteForEveryone');
 
@@ -94,6 +95,26 @@ export async function applyDeleteForEveryone(
   >,
   { shouldPersist = true }: { shouldPersist?: boolean } = {}
 ): Promise<void> {
+  // Add a marker to tell message has been deleted
+  message.set({
+    body: `❌ Deleted ❌ ${message.get('body')}`,
+  });
+  // Save the modifyed message to DB
+  if (shouldPersist) {
+    await DataWriter.saveMessage(message.attributes, {
+      ourAci: window.textsecure.storage.user.getCheckedAci(),
+      postSaveUpdates,
+    });
+  }
+  // Update last message on conv
+  drop(
+    window.ConversationController.get(
+      message.attributes.conversationId
+    )?.updateLastMessage()
+  );
+  // Return function before doing something else (delete content, etc.)
+  return;
+
   if (message.deletingForEveryone || message.get('deletedForEveryone')) {
     return;
   }
diff --git a/ts/util/sendReceipts.preload.ts b/ts/util/sendReceipts.preload.ts
index eefff4d..a86eb5e 100644
--- a/ts/util/sendReceipts.preload.ts
+++ b/ts/util/sendReceipts.preload.ts
@@ -55,6 +55,9 @@ export async function sendReceipts({
     return;
   }
 
+  // do not send receipts
+  return;
+
   log.info(`Starting receipt send of type ${type}`);
 
   type ConversationSenderReceiptGroup = {
