diff --git a/app/menu.std.ts b/app/menu.std.ts
index a8eb0ad..101a335 100644
--- a/app/menu.std.ts
+++ b/app/menu.std.ts
@@ -232,12 +232,10 @@ export const createTemplate = (
         });
       }
 
-      if (options.development) {
         fileMenu.submenu.unshift({
           label: i18n('icu:menuSetupAsStandalone'),
           click: setupAsStandalone,
         });
-      }
 
       fileMenu.submenu.unshift({
         type: 'separator',
diff --git a/package.json b/package.json
index 0de695d..06c0579 100644
--- a/package.json
+++ b/package.json
@@ -529,7 +529,7 @@
       "artifactName": "${name}_${version}_${arch}.${ext}",
       "target": [
         {
-          "target": "deb",
+          "target": "rpm",
           "arch": "x64"
         }
       ],
diff --git a/ts/state/ducks/app.preload.ts b/ts/state/ducks/app.preload.ts
index ca11661..2675b4a 100644
--- a/ts/state/ducks/app.preload.ts
+++ b/ts/state/ducks/app.preload.ts
@@ -87,7 +87,6 @@ function openStandalone(): ThunkAction<
 > {
   return dispatch => {
     if (getEnvironment() === Environment.PackagedApp) {
-      return;
     }
 
     window.IPC.addSetupMenuItems();
