diff --git a/app/menu.std.ts b/app/menu.std.ts
index a8eb0ad..101a335 100644
--- a/app/menu.std.ts
+++ b/app/menu.std.ts
@@ -232,12 +232,10 @@ export const createTemplate = (
         });
       }
 
-      if (options.development) {
         fileMenu.submenu.unshift({
           label: i18n('icu:menuSetupAsStandalone'),
           click: setupAsStandalone,
         });
-      }
 
       fileMenu.submenu.unshift({
         type: 'separator',
diff --git a/ts/state/ducks/app.preload.ts b/ts/state/ducks/app.preload.ts
index 4c0a0f9..2e9b986 100644
--- a/ts/state/ducks/app.preload.ts
+++ b/ts/state/ducks/app.preload.ts
@@ -89,7 +89,6 @@ function openStandalone(
       log.warn(
         `openStandalone: refusing because environment is ${getEnvironment()}`
       );
-      return;
     }
 
     cancelInstall()(dispatch, getState, undefined);
