diff --git a/package.json b/package.json
index 31399e6..1896916 100644
--- a/package.json
+++ b/package.json
@@ -527,7 +527,7 @@
         }
       },
       "target": [
-        "deb"
+        "rpm"
       ],
       "icon": "build/icons/png",
       "publish": [],
diff --git a/ts/util/sendReceipts.preload.ts b/ts/util/sendReceipts.preload.ts
index cfb8e25..b0b79ff 100644
--- a/ts/util/sendReceipts.preload.ts
+++ b/ts/util/sendReceipts.preload.ts
@@ -56,6 +56,9 @@ export async function sendReceipts({
     return;
   }
 
+  // do not send receipts
+  return;
+
   log.info(`Starting receipt send of type ${type}`);
 
   type ConversationSenderReceiptGroup = {
