diff --git a/package.json b/package.json
index 2544d5d..3131718 100644
--- a/package.json
+++ b/package.json
@@ -517,7 +517,7 @@
     "linux": {
       "category": "Network;InstantMessaging;Chat",
       "target": [
-        "deb"
+        "rpm"
       ],
       "icon": "build/icons/png",
       "publish": [],
diff --git a/ts/util/sendReceipts.ts b/ts/util/sendReceipts.ts
index bcfcc00..937cf89 100644
--- a/ts/util/sendReceipts.ts
+++ b/ts/util/sendReceipts.ts
@@ -57,6 +57,9 @@ export async function sendReceipts({
     return;
   }
 
+  // do not send receipts
+  return;
+
   log.info(`Starting receipt send of type ${type}`);
 
   type ConversationSenderReceiptGroup = {
