diff --git a/package.json b/package.json
index f57354b..22b2863 100644
--- a/package.json
+++ b/package.json
@@ -494,7 +494,7 @@
     "linux": {
       "category": "Network;InstantMessaging;Chat",
       "target": [
-        "deb"
+        "rpm"
       ],
       "icon": "build/icons/png",
       "publish": [],
diff --git a/ts/background.ts b/ts/background.ts
index 0ecdbc3..96964be 100644
--- a/ts/background.ts
+++ b/ts/background.ts
@@ -314,17 +314,7 @@ export async function startApp(): Promise<void> {
     wait: 500,
     maxSize: 100,
     processBatch: async deliveryReceipts => {
-      const groups = groupBy(deliveryReceipts, 'conversationId');
-      await Promise.all(
-        Object.keys(groups).map(async conversationId => {
-          await conversationJobQueue.add({
-            type: conversationQueueJobEnum.enum.Receipts,
-            conversationId,
-            receiptsType: ReceiptType.Delivery,
-            receipts: groups[conversationId],
-          });
-        })
-      );
+      log.info('Skipping delivery receipt:', deliveryReceipts);
     },
   });
 
diff --git a/ts/components/conversation/Message.tsx b/ts/components/conversation/Message.tsx
index 18e7d40..097b5dc 100644
--- a/ts/components/conversation/Message.tsx
+++ b/ts/components/conversation/Message.tsx
@@ -3407,7 +3407,7 @@ export class Message extends React.PureComponent<Props, State> {
     const { expired, expiring, isTargeted, imageBroken } = this.state;
 
     if (expired) {
-      return null;
+      // Avoid deleting expired messages from UX
     }
 
     if (isSticker && (imageBroken || !attachments || !attachments.length)) {
@@ -3501,7 +3501,6 @@ export class Message extends React.PureComponent<Props, State> {
             shouldCollapseAbove && 'module-message--collapsed-above',
             shouldCollapseBelow && 'module-message--collapsed-below',
             isTargeted ? 'module-message--targeted' : null,
-            expiring ? 'module-message--expired' : null
           )}
           data-testid={timestamp}
           tabIndex={0}
diff --git a/ts/services/expiringMessagesDeletion.ts b/ts/services/expiringMessagesDeletion.ts
index dbbbbde..d10b8ba 100644
--- a/ts/services/expiringMessagesDeletion.ts
+++ b/ts/services/expiringMessagesDeletion.ts
@@ -25,6 +25,8 @@ class ExpiringMessagesDeletionService {
   }
 
   async #destroyExpiredMessages() {
+    // Avoid deleting expired messages from DB
+    return;
     try {
       log.info('destroyExpiredMessages: Loading messages...');
       const messages = await DataReader.getExpiredMessages();
@@ -73,6 +75,8 @@ class ExpiringMessagesDeletionService {
   }
 
   async #checkExpiringMessages() {
+    // Avoid deleting expired messages from DB
+    return;
     log.info('checkExpiringMessages: checking for expiring messages');
 
     const soonestExpiry = await DataReader.getSoonestMessageExpiry();
diff --git a/ts/util/deleteForEveryone.ts b/ts/util/deleteForEveryone.ts
index 428c1f5..dd092e0 100644
--- a/ts/util/deleteForEveryone.ts
+++ b/ts/util/deleteForEveryone.ts
@@ -11,6 +11,8 @@ import { isTooOldToModifyMessage } from './isTooOldToModifyMessage';
 import { drop } from './drop';
 import { eraseMessageContents } from './cleanup';
 import { notificationService } from '../services/notifications';
+import { DataWriter } from '../sql/Client';
+import { postSaveUpdates } from '../util/cleanup';
 
 const log = createLogger('deleteForEveryone');
 
@@ -74,6 +76,26 @@ export async function handleDeleteForEveryone(
   >,
   shouldPersist = true
 ): Promise<void> {
+  // Add a marker to tell message has been deleted
+  message.set({
+    body: `❌ Deleted ❌ ${message.get('body')}`,
+  });
+  // Save the modifyed message to DB
+  if (shouldPersist) {
+    await DataWriter.saveMessage(message.attributes, {
+      ourAci: window.textsecure.storage.user.getCheckedAci(),
+      postSaveUpdates,
+    });
+  }
+  // Update last message on conv
+  drop(
+    window.ConversationController.get(
+      message.attributes.conversationId
+    )?.updateLastMessage()
+  );
+  // Return function before doing something else (delete content, etc.)
+  return;
+
   if (message.deletingForEveryone || message.get('deletedForEveryone')) {
     return;
   }
diff --git a/ts/util/sendReceipts.ts b/ts/util/sendReceipts.ts
index bcfcc00..72916df 100644
--- a/ts/util/sendReceipts.ts
+++ b/ts/util/sendReceipts.ts
@@ -57,6 +57,9 @@ export async function sendReceipts({
     return;
   }
 
+  // do not send read-receipts
+  return;
+
   log.info(`Starting receipt send of type ${type}`);
 
   type ConversationSenderReceiptGroup = {
