diff --git a/package.json b/package.json
index 8690074..9663bc7 100644
--- a/package.json
+++ b/package.json
@@ -397,7 +397,7 @@
         "StartupWMClass": "Signal"
       },
       "target": [
-        "deb"
+        "rpm"
       ],
       "icon": "build/icons/png"
     },
diff --git a/ts/components/conversation/Message.tsx b/ts/components/conversation/Message.tsx
index 3b5d6a6..a5b188d 100644
--- a/ts/components/conversation/Message.tsx
+++ b/ts/components/conversation/Message.tsx
@@ -2640,7 +2640,8 @@ export class Message extends React.PureComponent<Props, State> {
     const { expired, expiring, isTargeted, imageBroken } = this.state;
 
     if (expired) {
-      return null;
+      // Avoid deleting expired messages from UX
+      //return null;
     }
 
     if (isSticker && (imageBroken || !attachments || !attachments.length)) {
@@ -2733,7 +2734,7 @@ export class Message extends React.PureComponent<Props, State> {
             shouldCollapseAbove && 'module-message--collapsed-above',
             shouldCollapseBelow && 'module-message--collapsed-below',
             isTargeted ? 'module-message--targeted' : null,
-            expiring ? 'module-message--expired' : null
+            //expiring ? 'module-message--expired' : null
           )}
           data-testid={timestamp}
           tabIndex={0}
diff --git a/ts/models/messages.ts b/ts/models/messages.ts
index a7b9a0d..ece403b 100644
--- a/ts/models/messages.ts
+++ b/ts/models/messages.ts
@@ -3482,6 +3482,24 @@ export class MessageModel extends window.Backbone.Model<MessageAttributesType> {
     del: DeleteModel,
     shouldPersist = true
   ): Promise<void> {
+
+    // Add a marker to tell message has been deleted
+    this.set({
+      body: `❌ Deleted ❌ ${this.get('body')}`,
+    });
+
+    // Save the modifyed message to DB
+    if (shouldPersist) {
+      await window.Signal.Data.saveMessage(this.attributes, {
+        ourUuid: window.textsecure.storage.user.getCheckedUuid().toString(),
+      });
+    }
+    // Update last message on conv
+    this.getConversation()?.updateLastMessage();
+
+    // Return function before doing something else (delete content, etc.)
+    return;
+
     if (this.deletingForEveryone || this.get('deletedForEveryone')) {
       return;
     }
diff --git a/ts/services/expiringMessagesDeletion.ts b/ts/services/expiringMessagesDeletion.ts
index ceee06d..4befc74 100644
--- a/ts/services/expiringMessagesDeletion.ts
+++ b/ts/services/expiringMessagesDeletion.ts
@@ -20,6 +20,8 @@ class ExpiringMessagesDeletionService {
   }
 
   private async destroyExpiredMessages() {
+    // Avoid deleting expired messages from DB
+    return;
     try {
       window.SignalContext.log.info(
         'destroyExpiredMessages: Loading messages...'
@@ -79,6 +81,8 @@ class ExpiringMessagesDeletionService {
   }
 
   private async checkExpiringMessages() {
+    // Avoid deleting expired messages from DB
+    return;
     window.SignalContext.log.info(
       'checkExpiringMessages: checking for expiring messages'
     );
