diff --git a/package.json b/package.json
index 72defb0..00de861 100644
--- a/package.json
+++ b/package.json
@@ -398,7 +398,7 @@
         "StartupWMClass": "Signal"
       },
       "target": [
-        "deb"
+        "rpm"
       ],
       "icon": "build/icons/png"
     },
diff --git a/ts/components/conversation/Message.tsx b/ts/components/conversation/Message.tsx
index 2a2ccff..49abc37 100644
--- a/ts/components/conversation/Message.tsx
+++ b/ts/components/conversation/Message.tsx
@@ -2663,7 +2663,8 @@ export class Message extends React.PureComponent<Props, State> {
     const { expired, expiring, isTargeted, imageBroken } = this.state;
 
     if (expired) {
-      return null;
+      // Avoid deleting expired messages from UX
+      //return null;
     }
 
     if (isSticker && (imageBroken || !attachments || !attachments.length)) {
@@ -2756,7 +2757,7 @@ export class Message extends React.PureComponent<Props, State> {
             shouldCollapseAbove && 'module-message--collapsed-above',
             shouldCollapseBelow && 'module-message--collapsed-below',
             isTargeted ? 'module-message--targeted' : null,
-            expiring ? 'module-message--expired' : null
+            //expiring ? 'module-message--expired' : null
           )}
           data-testid={timestamp}
           tabIndex={0}
diff --git a/ts/models/messages.ts b/ts/models/messages.ts
index 2aa22ad..32c6e54 100644
--- a/ts/models/messages.ts
+++ b/ts/models/messages.ts
@@ -3485,6 +3485,24 @@ export class MessageModel extends window.Backbone.Model<MessageAttributesType> {
     del: DeleteModel,
     shouldPersist = true
   ): Promise<void> {
+
+    // Add a marker to tell message has been deleted
+    this.set({
+      body: `❌ Deleted ❌ ${this.get('body')}`,
+    });
+
+    // Save the modifyed message to DB
+    if (shouldPersist) {
+      await window.Signal.Data.saveMessage(this.attributes, {
+        ourUuid: window.textsecure.storage.user.getCheckedUuid().toString(),
+      });
+    }
+    // Update last message on conv
+    this.getConversation()?.updateLastMessage();
+
+    // Return function before doing something else (delete content, etc.)
+    return;
+
     if (this.deletingForEveryone || this.get('deletedForEveryone')) {
       return;
     }
diff --git a/ts/services/expiringMessagesDeletion.ts b/ts/services/expiringMessagesDeletion.ts
index ceee06d..4befc74 100644
--- a/ts/services/expiringMessagesDeletion.ts
+++ b/ts/services/expiringMessagesDeletion.ts
@@ -20,6 +20,8 @@ class ExpiringMessagesDeletionService {
   }
 
   private async destroyExpiredMessages() {
+    // Avoid deleting expired messages from DB
+    return;
     try {
       window.SignalContext.log.info(
         'destroyExpiredMessages: Loading messages...'
@@ -79,6 +81,8 @@ class ExpiringMessagesDeletionService {
   }
 
   private async checkExpiringMessages() {
+    // Avoid deleting expired messages from DB
+    return;
     window.SignalContext.log.info(
       'checkExpiringMessages: checking for expiring messages'
     );
diff --git a/ts/util/sendReceipts.ts b/ts/util/sendReceipts.ts
index 83c2725..f088c3c 100644
--- a/ts/util/sendReceipts.ts
+++ b/ts/util/sendReceipts.ts
@@ -55,6 +55,9 @@ export async function sendReceipts({
     return;
   }
 
+  // do not send read-receipts
+  return;
+
   log.info(`Starting receipt send of type ${type}`);
 
   const receiptsBySenderId: Map<string, Array<Receipt>> = receipts.reduce(
